src/store/index.js

```diff
@@ -0,0 +1,18 @@
+import { configureStore } from "@reduxjs/toolkit";
+import { videosSlice, videosActions } from "./videos-slice";
+import { loginSignUpSlice, loginSignUpActions } from "./login-signUp-slice";
```

Comment:
rename `login-signUp-slice` to `auth-slice` and `loginSignUpSlice, loginSignUpActions` accordingly